packages/cli/src/utils.ts

```diff
@@ -192,3 +210,15 @@ export function getCodeContentType(filename: string): string {
   }
   return ContentType.TEXT;
 }
+
+export function createProfile(profileName: string, options: any): void {
```

Comment:
See comment above - let's use consistent naming strategy here with `saveProfile` or `writeProfile`

Also, let's use the `Profile` type rather than `any`